src/mmrelay/plugins/nodes_plugin.py

```
@@ -87,16 +90,40 @@ def generate_response(self):
+            packet: Raw Meshtastic packet data received from the mesh.
+            formatted_message (str): Human-readable representation of the packet payload.
+            longname (str): Full device name of the packet sender.
+            meshnet_name (str): Name of the mesh network the packet originated from.
+
+        Returns:
+            `False` indicating the plugin did not handle the message.
+        """
         return False
 
-    async def handle_room_message(self, room, event, full_message):
+    async def handle_room_message(self, room, event, full_message) -> bool:
```

Ruff (0.14.6)

Unused method argument: `full_message`

(ARG002)